src/object.rs

```diff
@@ -1291,6 +1292,21 @@ pub trait ObjectExt: ObjectType {
     fn find_property<'a, N: Into<&'a str>>(&self, property_name: N) -> Option<::ParamSpec>;
     fn list_properties(&self) -> Vec<::ParamSpec>;
 
+    /// # Safety
+    ///
+    /// This function doesn't store type information
+    unsafe fn set_qdata<QD: 'static + Send + Sync>(&self, key: Quark, value: QD);
```

Comment:
If it's unsafe you can also remove the `Send` and `Sync` bounds :) For all GTK widgets it would be correct without, for example.

Reply:
Removed